### ISSUE TITLE
each domain should, after start, write some start message to the logs

### ISSUE THREAD

**Issue by calestyo** [dCache/dcache#3739]
Hi.

I just noticed this when starting up our production dCache at 3.2.8... (Debian with systemd):

In the old days ;-) ... the /var/log/dcache/domain.log were immediately created and got some start line that the domain was started...
I guess this was done by the startup scripts (i.e. the shell scripts) and not the domain itself.

Now with systemd this seems to be no longer the case... it's not a big issue... but it just feels strange to see nothing that confirms "oh, I'm running btw".

Also, the log files wouldn't get created by rsyslogd until something is written, so it looks even more as only half of the domains are running.

Cheers,
Chris

**Comment by paulmillar** [comment]
My feeling is that your expectation here is really just a hang-over from earlier behaviour.

With the change to systemd, systemctl is your "source of true" of whether or not a service is running.  Rather than looking at the log files for some feedback that the services are running, you should be running systemctl to get the same kind of feedback.  The log files are where to look when you know (or suspect) something is going wrong.

That said, we could add a message (for example) to say when all services within a domain have been started.  This would (indirectly) address the issue, since it would guarantee that something would be logged within a reasonable time.

**Comment by calestyo** [comment]
Yes, I though about something like this... which be quite handy for debugging as well. I.e. seeing when the domain was really actually running.
Sure systemd logs this as well (i.e. when it thinks it starts a unit), but in other logs and it may/may not match whit the actual startup.

**Comment by paulmillar** [comment]
The problem here is that, when starting a domain, some initialisation activity takes place after the cell is created -- the pool being a good example: the message could say dCache has started, except the pool is busy scanning the repository.  So it's not terribly clear what exactly is meant by being "ready".

However, we can add something and try to improve upon it based on people's feedback.

**Comment by calestyo** [comment]
Much easier:
Why not write e.g. the FQ domain name, the service (e.g. webdav) and perhaps the version just after the domain and perhaps the System cell starts, even before the actual service (and its cells) are started?
